lib/logstash/inputs/gelf.rb

```diff
@@ -29,9 +30,10 @@ class LogStash::Inputs::Gelf < LogStash::Inputs::Base
   # The IP address or hostname to listen on.
   config :host, :validate => :string, :default => "0.0.0.0"
 
-  # The port to listen on. Remember that ports less than 1024 (privileged
+  # The ports to listen on. Remember that ports less than 1024 (privileged
   # ports) may require root to use.
-  config :port, :validate => :number, :default => 12201
```

Comment:
I think we should keep the port option for two reasons:
1) most often than not you'll want to set both ports to the same
2) removing this option introduces a breaking change, which means a new major version. This should be avoided.

Reply:
The behaviour would then be:
a) whatever you set in `:port` will be used by both `:port_udp` and `:port_tcp`
b) setting `:port_udp` and/or `:port_tcp` overrides the value of `:port`

Reply:
Done by reintroducing port and setting @port_tcp and @port_udp to port if they are not set by the user

Reply:
Hmm I doint see the port. I only see port_udp and port_tcp :)

Reply:
Sorry. I pushed it now.